internal/pkg/postgres/issues.go

```diff
@@ -17,6 +17,64 @@ type issue struct {
 	EndTime     *time.Time
 }
 
+/*func (c *Client) inactiveIssueID(ctx context.Context, tx pgx.Tx, roomID string) (int, error) {
```

Comment:
Why is all of this commented out? Is it old code, or future code? If it's old code we ought to just delete it.